microsoft-edge/webview2/reference/javascript/index.yml

```diff
@@ -0,0 +1,18 @@
+### YamlMime:TSPackage
+uid: WebView2Script!
+metadata:
+  title: JavaScript APIs in WebView2
+name: WebView2Script
+type: package
```

Comment:
on the following page: 
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/reference/javascript/?branch=pr-en-us-2315

I see:
WebView2Script package
Reference

Suggestion we change it to 
Web pages can use these JavaScript APIs to communicate with WebView2 objects using these classes.